src/scss/tools/mixins/_border.scss

```diff
@@ -7,8 +7,12 @@
 //
 // @include bordered();
 
+$bordered-radius    : $border-radius !default;
```

Comment:
Not sure you'll be able to call the mixin and change these values by doing it this way? I was think more like:

```
@mixin bordered($bordered-radius: 1px, $bordered-spacing : 5px, $bordered-colour: red ) {
    border: 1px solid $bordered-colour;
    padding: $bordered-spacing;
    border-radius: $bordered-radius;
}

.foo {
@include bordered(5px);
}
```

Reply:
Totally - i confused myself with the word default 😄 - Updated!